modules/nw-nwop-viewing-config.adoc

```diff
@@ -0,0 +1,94 @@
+// Module filename: nw-nwop-viewing-config.adoc
+// Module included in the following assemblies:
+// * networking/configuring-network-operator.adoc
+
+[id='nw-nwop-viewing-config-{context}']
+= Viewing the Network Configuration in a Running Cluster
+
+
+
+.Prerequisites
+
+* Start a cluster and login using your credentials.
```

Comment:
log in

If we're turning this into an installation assembly, you should include the modules to install the oc cli.